### ISSUE TITLE
Get a config in xdg-miriway to disable the broken MATE panel elements

### ISSUE THREAD

**Issue by AlanGriffiths** [Miriway/Miriway#26]

**Comment by AlanGriffiths** [comment]
I suspect that mate-panel is getting its settings from `dconf`, which would mean fiddling with `DCONF_PROFILE` and  `$XDG_CONFIG_HOME/dconf/`.

**Comment by TheNathan0** [comment]
Great work here.  I don't know if anyone has noticed yet however, something changed (likely in the default MATE configuration I'm assuming) which renders the MATE panel unusable on Miriway I'm afraid.  After running the script and running Miriway am I met with a few errors surrounding invalid panel contents and two entirely blank panels.

**Comment by AlanGriffiths** [comment]
@TheNathan0 the errors you see are panel content that doesn't work in this environment, either because they don't work on Wayland, or because they depend on some part of MATE the script doesn't set up.

You should be able to right click on the panel and add content, some of which will work.

All of this really needs someone to who knows, or has time to dig into, the way that MATE desktop is set up. I've provided the script as a starting point, and am happy to accept improvements. But, until MATE has much better support for Wayland this won't be a priority for me.

**Comment by TheNathan0** [comment]
Makes sense.  I suspect that some change to MATE was pushed that broke the already limited Wayland support probably.  I've been unable to get mate-panel to display anything besides blankness.

**Comment by AlanGriffiths** [comment]
MATE panel seems to be working fine now
